src/main/java/rocks/nt/apm/jmeter/JMeterInfluxDBBackendListenerClient.java

```diff
@@ -101,12 +103,31 @@
 	 * Processes sampler results.
 	 */
 	public void handleSampleResults(List<SampleResult> sampleResults, BackendListenerContext context) {
+
+		// Indicates whether to write sub sample records to the database
+		String recordSubSamples = context.getParameter(KEY_RECORD_SUB_SAMPLES, "");
```

Comment:
Please create a **boolean** variable "recordSubSamples" that is initialized in the setupTest(....) method.